theia/operations/panoptes_operations/upload_subject.py

```diff
@@ -8,26 +8,21 @@
 
 from datetime import datetime
 
+
 class UploadSubject(AbstractOperation):
     def apply(self, filenames):
         if self.pipeline.multiple_subject_sets:
             scope = self.bundle
         else:
             scope = self.pipeline
 
-        self.authenticated_panoptes = Panoptes(
-             endpoint=PanoptesUtils.base_url(),
-             client_id=PanoptesUtils.client_id(),
-             client_secret=PanoptesUtils.client_secret()
+        authenticated_client = Panoptes(
```

Comment:
maybe clarify this is using the special server side oauth client app vs user supplied one
```suggestion
        server_authenticated_client = Panoptes(
```

Reply:
Renamed!